### PR TITLE
add support for preview images in list views

### PR DESCRIPTION
**Pull request by azplanlos**
I don't know if this small improvement is of interest for anyone else except me, but I added an option to include preview images in list views. By default every posts first image is taken as preview image and resized to a smaller size. If no image is present a feather icon can be used. Also in this PR taxonomy pages use the correct title as title instead of the hard coded title "Tags".

### CODE REVIEW

File: exampleSite/config.toml [hunk 1]
```diff
@@ -45,6 +45,7 @@ pygmentsStyle = "monokai"
     {name = "email", url = "mailto:example@example.com"}
   ]
   metaKeywords = ["blog", "gokarna", "hugo"]
+  ShowPreviewImages = "false"
```
Comment (yashmehrotra) [inline exampleSite/config.toml:48]: Please use boolean type not string

Comment (yashmehrotra) [reply exampleSite/config.toml:48]: Also, use smaller case and change variable to `enablePreviewImages`

**Review by yashmehrotra — CHANGES_REQUESTED** [review]

**Comment by yashmehrotra** [comment]
@azplanlos Hey, when I tried this on my local, this was not working as expected and the images were not showing up

Can you update your PR with the param as true in the `exampleSite/config.toml` file so that we can see a working example on our netlify preview

Currently it looks like this:

image

**Comment by 526avijitgupta** [comment]
@azplanlos Any updates here?

**Comment by 526avijitgupta** [comment]
Closing. Please reopen @azplanlos if there are any updates

**Comment by azplanlos** [comment]
Sorry, didn't see your messages. I will do the changes and provide an update.

**Pull request closed without merging**